CONTRIBUTING.md

```diff
@@ -0,0 +1,148 @@
+# LTTng-UST contributor's guide
+
+Being an open source project, the LTTng-UST project welcomes
+contributions from anyone. This guide walks you through the process
+of contributing a patch to LTTng-UST.
+
+
+## Getting the source code
+
+The LTTng-UST project uses [Git](https://git-scm.com/) for version
+control. The upstream Git repository URL is:
+
+    git://git.lttng.org/lttng-ust.git
+
+
+## Coding standard
+
+LTTng-UST uses the
+[Linux kernel coding style](http://www.kernel.org/doc/Documentation/CodingStyle)
+with one addition: single-line `if`/`for`/`while` statements must be
+wrapped in braces.
```

Comment:
That's what I thought. Please see the latest fixup (2097284) and use it if it addresses your comment suitably.